ntbs-service/Models/NtbsContext.cs

```diff
@@ -26,19 +26,15 @@ public NtbsContext(DbContextOptions<NtbsContext> options)
 
         public virtual DbSet<Country> Country { get; set; }
         public virtual DbSet<Ethnicity> Ethnicity { get; set; }
-        public virtual DbSet<TBService> TBService { get; set; }
```

Comment:
AFAIK this is already slightly inconsistent in the codebase so happy to go with whicever convention but worth noting what we want to use going forward :)